platform-sdk/platform-apps/demos/HashgraphDemo/src/main/java/com/swirlds/demo/hashgraph/HashgraphDemoMain.java

```diff
@@ -237,9 +237,9 @@ public void paintComponent(final Graphics g) {
             final double createCons = (double) metrics.getValue(PLATFORM_CATEGORY, "secC2C");
             final double recCons = (double) metrics.getValue(PLATFORM_CATEGORY, "secR2C");
 
-            print(g, "%5.0f trans/sec", (double) metrics.getValue(PLATFORM_CATEGORY, "trans/sec"));
-            print(g, "%5.0f events/sec", (double) metrics.getValue(PLATFORM_CATEGORY, "events/sec"));
-            print(g, "%4.0f%% duplicate events", (double) metrics.getValue(PLATFORM_CATEGORY, "dupEv%"));
+            print(g, "%5.0f trans_per_sec", (double) metrics.getValue(PLATFORM_CATEGORY, "trans_per_sec"));
```

Comment:
I wonder if making metric names constants in some sort of `EventMetricsConstants` class would be useful at some point.  This comment isn't meant to block this PR, just observing that a common constant would make future metric name changes occur everywhere at once.

Reply:
yes, I agree. But we would still need to find a place to put those constants compatible with the modularization efforts going on.